### PR TITLE
Master menu search 2 yti

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/xml/base.xml [hunk 1]
```diff
@@ -1341,7 +1341,7 @@
         <ul class="dropdown-menu" role="menu">
             <li><a href="#" data-menu="documentation">Documentation</a></li>
             <li><a href="#" data-menu="support">Support</a></li>
-            <li><a href="#" data-menu="about">About</a></li>
+            <li class="about"><a href="#" data-menu="about">About</a></li>
```
Comment (tivisse) [inline addons/web/static/src/xml/base.xml]: Before divider instead of after about

File: addons/mail/static/src/less/systray.less [hunk 2]
```diff
@@ -5,6 +5,9 @@
         > i {
             font-size: larger;
         }
+        &:hover, &:focus {
```
Comment (tivisse) [inline addons/mail/static/src/less/systray.less]: hover is already done

File: addons/web/static/src/less/navbar.less [hunk 3]
```diff
@@ -23,6 +23,7 @@
         }
         &:focus, &:active, &:focus:active {
```
Comment (tivisse) [inline addons/web/static/src/less/navbar.less]: To check but this commit seems uslesss

**Comment by tde-banana-odoo** [comment]
M&Ms ?

**Pull request merged**